src/openforms/js/components/admin/form_design/form-creation-form.js

```diff
@@ -187,7 +187,7 @@ function reducer(draft, action) {
      * Form-level actions
      */
     case 'BACKEND_DATA_LOADED': {
-      const {formData, supportingData} = action.payload;
```

Comment:
To be in line with:

https://github.com/open-formulieren/open-forms/blob/cc8fe709fcb40280c8d32bdab45f55b57304a7a0/src/openforms/js/components/admin/form_design/form-creation-form.js#L190

Reply:
I don't understand what this fixes

Reply:
This is to use the same order as defined [here](https://github.com/open-formulieren/open-forms/blob/cc8fe709fcb40280c8d32bdab45f55b57304a7a0/src/openforms/js/components/admin/form_design/form-creation-form.js#L979), less confusing imo

Reply:
okay - next time please try to avoid refactoring code that's not directly related to the issue as it clutters the diff, instead make a separate PR to clean that up. Small cleanup PRs like that can be made without issue :)